tests/oeedger8r/enc/CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ oeedl_file(
     --edl-search-dir ../moreedl
 )
 
+# Dummy target used for generating from EDL on demand.
+add_custom_target(edl_gen DEPENDS ${all_t})
```

Comment:
Not clear what this target does.

Reply:
From the commit:

> This is useful to force the (re)generation of the generated files for
the edger8r test EDL files without having to rebuild anything, extra
useful on systems where the rest of the project doesn't build, such as
my MacBook.